core/src/html/layout.rs

```diff
@@ -470,11 +470,15 @@ impl<'a, 'gc> LayoutContext<'a, 'gc> {
         // In an ideal world, device fonts would search for a matching font on the system and render it in some way.
         if !is_device_font {
             if let Some(font) = library
-                .get_font_by_name(&font_name, span.bold, span.italic)
+                .get_embedded_font_by_name(&font_name, span.bold, span.italic)
                 .filter(|f| f.has_glyphs())
             {
                 return Some(font);
             }
+            // TODO: If set to use embedded fonts and we couldn't find any matching font, show nothing
+            // However - at time of writing, we don't support DefineFont4. If we matched this behaviour,
+            // then a bunch of SWFs would just show no text suddenly.
```

Comment:
That's weird- I remember removing `DefineFont4` tags and the `Font.registerFont` calls, and TLF text still showed up.

Reply:
I'd have to check that swf and find out 😅 in his path it shouldn't use fallback fonts at all, and there's a test to confirm it